### ISSUE TITLE
Setting up apicurio with keycloak within a docker-compose environment

### ISSUE THREAD

**Issue by abryant710** [AndyYuen/securing-apicurio-service-registry#1]
From your experience do you know if it is possible to set up this environment within docker-compose? Something like the attached.

[docker-compose.yml.txt](https://github.com/AndyYuen/securing-apicurio-service-registry/files/11833251/docker-compose.yml.txt)
[import-certificate.sh.txt](https://github.com/AndyYuen/securing-apicurio-service-registry/files/11833252/import-certificate.sh.txt)
[Dockerfile.txt](https://github.com/AndyYuen/securing-apicurio-service-registry/files/11833250/Dockerfile.txt)

**Comment by AndyYuen** [comment]
@abryant710 Looks OK. The only way to be sure is to try it out. Have you?

**Comment by abryant710** [comment]
@AndyYuen , yes I have tried it out. Keycloak seems to work fine but the apicurio container is unable to communicate with keycloak in the internal network.

**Comment by AndyYuen** [comment]
I am afraid that I don't know the differences between the apicurio base image and the Red Hat images. Did you look at the log of the apicurio container? Does it give you any idea as to why it cannot communicate with keycloak? Connection refused? Not able to reach keycloak? Can it connect to the Postgres database?
